application/routes.js

```diff
@@ -862,7 +862,7 @@ module.exports = function (server) {
   //LTI
   server.route({
     method: 'POST',
-    path: '/lti/handle',
+    path: '/lti/handle/{resource_id}',
```

Comment:
Rethink this parameter (its not used atm) and fix it in the code - either remove it or use it.